doc/openapi-user-guide.md

```diff
@@ -114,7 +114,7 @@ metadata:
   name: my-secret
   namespace: 3scale-test
 data:
-  issuerEndpoint: https://3scale-zync:some-secret@keycloak-rhsso-test.apps.xxxxx.xxxx.s1.devshift.org/auth/realms/petstore
+  issuerEndpoint: https://3scale-zync:secret-name@keycloak-rhsso-test.example.com/auth/realms/petstore
```

Comment:
`secret-name`???? That can create confusion

Reply:
Do we agree that it is not about a kubernetes secret name, right?

Reply:
Yes, sure, it's confusion, mistake, in previous revision it was `some-secret`. Thank you very much!  Updated.